src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/HtmlHelperValidationExtensions.cs

```diff
@@ -20,7 +20,8 @@ public static class HtmlHelperValidationExtensions
         /// <param name="expression">Expression name, relative to the current model.</param>
         /// <returns>
         /// A new <see cref="IHtmlContent"/> containing a <see cref="ViewContext.ValidationMessageElement"/> element.
-        /// <c>null</c> if the <paramref name="expression"/> is valid and client-side validation is disabled.
+        /// <see cref="HtmlString.Empty"/> if the <paramref name="expression"/> is valid and client-side validation is
```

Comment:
I suspect that `HtmlString.Empty` is an implementation detail, so we could perhaps instead say `An empty IHtmlContent if the ...`